examples/sops/README.md

```diff
@@ -0,0 +1,46 @@
+# SOPS
+
+The `sops` config function transforms an input kpt package according 
+to the function configuration: in the current version it can decrypt
+the documents that have `sops` field with SOPS metadata.
+This example invokes the sops -d function using declarative 
+configuration. See [sops readme](https://github.com/mozilla/sops/blob/master/README.rst) for more details.
+
+In order to decrypt yaml sops may accept a variaty of ENV vars, e.g.
+to work with Hashicorp Vault it will be necessary to set: `VAULT_ADDR` and `VAULT_TOKEN`.
+For PGP case this function introduces `SOPS_IMPORT_PGP` ENV var that must
+contain the private key or keys needed to decrypt yamls. If you have a file with keys 
+it's possible to run:
+
+```
+SOPS_IMPORT_PGP=$(cat sops_functional_tests_key.asc) kpt fn run <folder>
+```
+
+or if your keys are already in gpg, it's possibe to run:
+
+```
+SOPS_IMPORT_PGP=$(gpg --armor --export-secret-keys) kpt fn run <folder>
+```
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/sops .
+SOPS_IMPORT_PGP=$(cat sops/sops_functional_tests_key.asc) kpt fn run sops/local-configs
+```
+
+## Expected result
+
+Verify the updated configuration:
+
+```sh
+kpt cfg cat sops/local-configs
+```
+
+## Note:
+
+Starting kustomize 3.8.1 this function can be also used by kustomize as a plugin, but
```

Comment:
I'd avoid this Note given this user guide is not concerned about kustomize binary.

Reply:
will remove it. Thank you for the review!